bentoml/_internal/server/__init__.py

```diff
@@ -109,6 +124,7 @@ def serve_development(
     )
 
     arbiter = create_standalone_arbiter(watchers)
+    _ensure_prometheus_dir()
```

Comment:
👍 great job refactoring this, it's clear this is an one time setup and not invoked in child process